ports/stm32f4/mpconfigport.h

```diff
@@ -40,6 +40,8 @@
 #include "py/circuitpy_mpconfig.h"
 
 #define MICROPY_PORT_ROOT_POINTERS \
+	/* pointers to all 10 UART objects (if they have been created) */ \
+    void *cpy_uart_obj_all[9]; \
```

Comment:
```suggestion
    void *cpy_uart_obj_all[10]; \
```

Maybe the mcu def should define how many their are?

Reply:
True. And I should expand the quantity to the full 10, which is the maximum possible supported via the F413

Reply:
mmm. I can't include peripherals/stm32f4/periph.h at this stage of compilation, however. And putting it in the board defines would mean a repeated define for every board that never changes. 

I'm going to leave it for now. But I do want to revisit this, because I think there's a lot of MCU stuff that's relegated to board definitions right now, and I think that might be appropriate to change.

Reply:
Why can't you include it here?